### ISSUE TITLE
Wormhole Bug: WBTC (ERC-20) to WBTC (SOL) stuck on "Awaiting completion on Solana..."

### ISSUE THREAD

**Issue by ralopibralop** [solana-labs/oyster#196]
Hey guys, please kindly check out this discussion for a bug, TXs, etc.:
https://github.com/certusone/wormhole/discussions/333

I was testing wormhole (before trying with a larger sum) and tried to move 0.0001 WBTC from ERC-20 to SOL. The process got stuck on "Awaiting completion on Solana..." and it looks like the destination token account was not created, but the lockup was.

Would appreciate it if you could create a destination token account, so I can ask to resubmit VAA.

Thanks a lot!

**Comment by leoluk** [comment]
@bartosz-lipinski This user needs to create their token account (iirc, this happened once before and there's a special one-off page to do it?)

**Comment by yamijuan** [comment]
It should be do-able from the UI. Select the token from ETH side, put a very very small amount of that token, and click transfer. 
The FIRST pop up should be on solana wallet asking to create an account. That one should be accepted. After that REJECT all metamask transactions.  That should only create an associated token account in solana. @leoluk

**Comment by leoluk** [comment]
No response - assuming it's solved

**Comment by misterbaka** [comment]
hi guys i have the same issue above.  i tried what yamijuan suggested but no luck.  

i tried another smaller transfer and that went through.

can you please advise its a large amount of coins to me.  thank you

https://github.com/solana-labs/oyster/issues/310

**Comment by leoluk** [comment]
@misterbaka Please stop hijacking existing issues

**Comment by Gitsomelearnsome** [comment]
Thanks for following up @leoluk , I just followed the steps proposed by @yamijuan 

Here is the approved Solana tx:
https://explorer.solana.com/tx/5Q22R53AJvNBcZwuzpoEPzd35Kvj9yh14BjkKExnzoVqF91ebKXP2KAhhUCU2NTnx5U9H8mXoa33Z67hQuNCNx4D

For clarity: It's re: this issue:
https://github.com/solana-labs/oyster/issues/305

I still don't have the ability to access my initial wLuna deposit. 

This section also stays empty:
![image](https://user-images.githubusercontent.com/40437233/133073381-13c94f6d-57e5-4183-bf08-70ffa978fce4.png)


It does seem to have registered a Solana account since a new transaction does not prompt that request anymore, but goes straight to approving ethereum txs (which I've rejected). 

Help appreciated.
